Main.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Unit.h"
+#include "Control.h"
+
+int main(int argc,char** argv)
+{
+	Control vezerles;
+	std::string player_name = argv[1];
+	int hp = std::atoi(argv[2]);
+	int dmg = std::atoi(argv[3]);
+	Unit player1(hp,dmg,player_name);
+	player_name = argv[4];
+	hp = std::atoi(argv[5]);
+	dmg = std::atoi(argv[6]);
+	Unit player2(hp,dmg,player_name);
+	
+	
+	vezerles.Attack(player1,player2);

```

Comment:
Oke, azt mondtatok, hogy marad a vezerles osztlay, de most akkor kerlek ird meg, miert jobb ez igy, mintha csak siman lenne mondjuk egy standalona attack fuggvenyed? Mitol lett jobb, hogy beraktad egy osztalyba?

Reply:
Azzal a feltételezéssel hoztuk létre a Vezérlés/Control osztályt, hogy a jövőben még több, karakterek közötti interakciót megvalósító metódust is realizálni kell. Ezeknek a metódusoknak lenne egyfajta gyűjtőhelye a vezérlés osztály. Innen igazából számunkra átláthatóbb a kód, mintha a main-be lenne belehányva össze vissza mindenféle függvény/metódus.

Reply:
Hat egyelore egy for ciklus kellene csak a main-be, igazabol mar a fuggveny is overkill, de jol van, nem eroltetem :-) Maximum tobb munkatok lesz a jovoben,